server/src/main/webapp/WEB-INF/rails/webpack/views/pages/agent-job-run-history/agent_job_state_transitions_modal.tsx

```diff
@@ -37,14 +37,43 @@ export class AgentJobStateTransitionModal extends Modal {
 
     return <div data-test-id={`job-state-transitions-for-${this.jobRepresentation()}`}>
       <div class={styles.jobStateTransitionInformationContainer} data-test-id="additional-information-container">
-        <KeyValuePair data={new Map(
-          [
-            ["Job", this.jobRepresentation()],
-            ["Wait Time", `${this.findTimeDifference("Scheduled", "Assigned")}`],
-            ["Building Time", `${this.findTimeDifference("Assigned", "Completed")}`],
-            ["Total Time", `${this.findTimeDifference("Scheduled", "Completed")}`]
-          ])
-        }/>
+
+        <div data-test-id="key-value-pair" class={styles.keyValuePair}>
+          <div class={styles.key} data-test-id="key-job">Job</div>
+          <span class={styles.value} data-test-id="value-job">
+            : {this.jobRepresentation()}
+          </span>
+        </div>
+
+        <div data-test-id="key-value-pair" class={styles.keyValuePair}>
+          <div class={styles.key} data-test-id="key-wait-time">
+            Wait Time
+            <Tooltip.Info content={"Wait Time is the time spent by the job waiting for an agent."}/>
+          </div>
+          <span class={styles.value} data-test-id="value-wait-time">
+            : {this.findTimeDifference("Scheduled", "Assigned")}
+          </span>
+        </div>
+
+        <div data-test-id="key-value-pair" class={styles.keyValuePair}>
+          <div class={styles.key} data-test-id="key-building-time">
+            Building Time
+            <Tooltip.Info content={"Building time is the time spent building the job on an agent."}/>
+          </div>
+          <span class={styles.value} data-test-id="value-building-time">
+            : {this.findTimeDifference("Assigned", "Completed")}
+          </span>
+        </div>
+
+        <div data-test-id="key-value-pair" class={styles.keyValuePair}>
+          <div class={styles.key} data-test-id="key-total-time">
+            Total Time
+            <Tooltip.Info content={"Total time is the time taken by the job from scheduling to completion. This is the total time spent by the job in waiting and building state."}/>
```

Comment:
@adityasood @maheshp -- does this description sound right? is there a better way to phrase this?